js/piwik.js

```diff
@@ -2439,7 +2439,9 @@ if (typeof window.Matomo !== 'object') {
                 uniqueTrackerId = trackerIdCounter++,
 
                 // whether a tracking request has been sent yet during this page view
-                hasSentTrackingRequestYet = false;
+                hasSentTrackingRequestYet = false,
+
+                fingerPrintEnable = true;
```

Comment:
We usually prefix these variables with `config` see eg `configConsentRequired` or `configCookiesToDelete`